Project/Assets/Editor/PrairieGUI/TwineNodeInspector.cs

```diff
@@ -20,5 +26,34 @@ public override void OnInspectorGUI ()
 		parentArray = PrairieGUI.drawObjectList ("Parents", parentArray);
 		node.parents = new List<GameObject> (parentArray);
 
+		// For the node to be a decision node, we'll say it has 
+		//	to have a "prairie_decision" tag, AND it has to have 
+		//	the isDecisionNode checkbox checked.
+		bool hasDecisionTag = node.tags.Contains (TwineNode.PRAIRIE_DECISION_TAG);
```

Comment:
But I'm still going to streamline it since we're going to get rid of tags in the TwineNode inspector